py3status/modules/speedtest_cli.py

```diff
@@ -0,0 +1,302 @@
+# -*- coding: utf-8 -*-
+"""
+Do a bandwidth test with speedtest-cli.
+
+Configuration parameters:
+    button_refresh: button to run the check (default 1)
+    button_share: button to open the result link (default None)
+    format: display format for this module, {download} and/or {upload} required
```

Comment:
>{download} and/or {upload} required

Why? Bad code? Users should be able to skip this if they want to. :laughing:

Reply:
Maybe I missunderstand your remark. 
I have done a system which autoswitch speedtest-cli based on params. 
But yes, I can run basic speedtest-cli with download AND upload if no download or upload placeholders used.